pyacq/devices/npbufferdevice.py

```diff
@@ -40,14 +40,12 @@ def _configure(self, nb_channel=16, sample_interval=0.001, chunksize=256,
         self.chunksize = chunksize
         self.timeaxis = timeaxis
         
-        
```

Comment:
My vispy PR has some extensive changes to this node that make it a bit more generic. Perhaps we should merge that before making these changes (otherwise the conflicts will be a pain to resolve)?